docs/_source/docs/templates.rst

```diff
@@ -31,6 +31,121 @@ Template. Sceptre User Data is accessible within Templates as
+
+.. code-block:: yaml
+
+    AWSTemplateFormatVersion: '2010-09-09'
+    Description: 'Add Route53 - CNAME and ALIAS records'
+    Parameters:
+      DomainName:
+        Type: String
+        Default: example.net
+      Zone:
+        Type: String
+      {% if sceptre_user_data.CNAMErecords is defined %}{% for rule in sceptre_user_data.CNAMErecords %}
+      {{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}cnamerecord:
+        Type: String
+        Default: "{{rule.record}}"{% endfor %}{% endif %}
+      {% if sceptre_user_data.ALIASrecords is defined %}{% for rule in sceptre_user_data.ALIASrecords %}
+      {{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}aliasrecord:
+        Type: String
+        Default: "{{rule.record}}"
+      {{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}aliasvalue:
+        Type: String
+        Default: "{{rule.dnsnamealias}}"
+      {{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}aliaszoneid:
+        Type: String
+        Default: "{{rule.hostedzoneid}}"
+      {% endfor %}{% endif %}
+    Resources:
+    {% if sceptre_user_data.CNAMErecords is defined %}{% for rule in sceptre_user_data.CNAMErecords %}add{{ rule.record |    replace("-","d")|replace("_","s")|replace('.',"p")}}cnamerecord:
+        {% set record = rule.record %}
+        Type: 'AWS::Route53::RecordSet'
+        Properties:
+          Name: !Join
+            - ""
+            - [ !Sub '${ {{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}cnamerecord }','.', !Ref DomainName, '.']
+          HostedZoneId: !Sub '${Zone}'
+          Type: CNAME
+          TTL: {{ rule.ttl }}
+          ResourceRecords:
+            - {{ rule.address }}
+      {% endfor %}{% endif %}
+        {% if sceptre_user_data.ALIASrecords is defined %}{% for rule in sceptre_user_data.ALIASrecords %}
+      {% set entry = rule.record |replace("-","d")|replace("_","s")|replace('.',"p")%}add{{entry}}aliasrecord:
+        Type: AWS::Route53::RecordSet
+        Properties:
+          {% if rule.record == "" %}
+          Name: !Ref DomainName
+          {% else %}
+          Name: !Join
+            - ""
+            - [ !Sub '${ {{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}aliasrecord }','.', !Ref DomainName, '.']
+          {% endif %}
+          Type: A
+          HostedZoneId: !Ref Zone
+          AliasTarget:
+            DNSName: "{{ rule.dnsnamealias }}"
+            HostedZoneId: "{{ rule.hostedzoneid }}"
+      {% endfor %}{% endif %}
+    Outputs:
+      {% if sceptre_user_data.CNAMErecords is defined %}{% for rule in sceptre_user_data.CNAMErecords %}add{{ rule.record |    replace("-","d")|replace("_","s")|replace('.',"p")}}cnamerecord:
+        Value: !Ref 'add{{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}cnamerecord'
+        # Description: 'add{{ rule.record |replace("-","d")|replace("_","s")|replace('.',"p")}}cnamerecord'
```

Comment:
is this deliberately commented out?

Reply:
no, this was a mistake. Thanks for noticing :smile: